taskcluster/tc-train-unittests.sh

```diff
@@ -4,21 +4,14 @@ set -xe
 
 source $(dirname "$0")/tc-tests-utils.sh
 
-extract_python_versions "$1" "pyver" "pyver_pkg" "py_unicode_type" "pyconf" "pyalias"
```

Comment:
Pretty sure you still need this, it's what defines the pyalias variable used below.

Reply:
ohh deleted it by mistake!